docs/source/minimal_example.rst

```diff
@@ -44,15 +44,15 @@ everest_config.yml::
 
 **Details**
 
-``name`` control defined name
+``name``: control defined name
 
-``min`` minimal value each of the control variables can take
+``min``: minimal value each of the control variables can take
 
-``max`` maximal value each of the control variables can take
+``max``: maximal value each of the control variables can take
 
-``initial_guess`` the initial value for the control variables
+``initial_guess``: the initial value for the control variables
 
-``min``, ``max`` and ``initial_guess`` can also be defined for each control individually::
```

Comment:
The `::` double colons are needed to mark the start of the literal block that follows, the should remain as is.